audit-service/src/unit-tests/java/uk/gov/gchq/palisade/service/audit/service/StroomAuditServiceTest.java

```diff
@@ -73,233 +58,147 @@ public void setUp() {
-
-        assertAll(
-                () -> assertThat(logCaptor.getAllValues().contains(userId.getId())),
-                () -> assertThat(logCaptor.getAllValues().contains(context.getPurpose())),
-                () -> assertThat(logCaptor.getAllValues().contains(requestId.getId())),
-                () -> assertThat(logCaptor.getAllValues().contains(StroomAuditService.REGISTER_REQUEST_NO_RESOURCES_TYPE_ID)),
-                () -> assertThat(logCaptor.getAllValues().contains(StroomAuditService.REGISTER_REQUEST_NO_RESOURCES_DESCRIPTION)),
-                () -> assertThat(logCaptor.getAllValues().contains(StroomAuditService.REGISTER_REQUEST_NO_RESOURCES_OUTCOME_DESCRIPTION))
-        );
-    }
-
-    @Test
-    public void auditRegisterRequestSuccessful() {
-        // Given
-        final AuditRequest auditRequest = AuditRequest.RegisterRequestCompleteAuditRequest.create(requestId)
-                .withUser(user)
-                .withLeafResources(Sets.newSet(resource))
-                .withContext(context);
-
-        // When
-        auditService.audit(auditRequest);
-
-        //Then
-        verify(eventLogger, atLeastOnce()).log(logCaptor.capture());
-        final String log = eventSerializer.serialize(logCaptor.getValue());
-
-        assertAll(
-                () -> assertThat(logCaptor.getAllValues().contains(userId.getId())),
-                () -> assertThat(logCaptor.getAllValues().contains(context.getPurpose())),
-                () -> assertThat(logCaptor.getAllValues().contains(requestId.getId())),
-                () -> assertThat(logCaptor.getAllValues().contains(resource.getId())),
-                () -> assertThat(logCaptor.getAllValues().contains(resource.getType())),
-                () -> assertThat(logCaptor.getAllValues().contains(StroomAuditService.REGISTER_REQUEST_COMPLETED_TYPE_ID)),
-                () -> assertThat(logCaptor.getAllValues().contains(StroomAuditService.REGISTER_REQUEST_COMPLETED_DESCRIPTION))
-        );
     }
 
     @Test
-    public void auditRegisterRequestUserException() {
+    void testDataServiceAuditSuccessMessage() {
         // Given
-        final AuditRequest auditRequest = AuditRequest.RegisterRequestExceptionAuditRequest.create(requestId)
-                .withUserId(userId)
-                .withResourceId(resource.getId())
-                .withContext(context)
-                .withException(exception)
-                .withServiceName(ServiceName.USER_SERVICE.name());
-        auditRequest.setOriginalRequestId(requestId);
 
         // When
-        auditService.audit(auditRequest);
+        auditService.audit(ApplicationTestData.TEST_TOKEN, ApplicationTestData.auditSuccessMessage(ServiceName.DATA_SERVICE.value));
 
         //Then
         verify(eventLogger, atLeastOnce()).log(logCaptor.capture());
-        final String log = eventSerializer.serialize(logCaptor.getValue());
+        final String log = EVENT_SERIALIZER.serialize(logCaptor.getValue());
```

Comment:
log is unused in these tests

Reply:
I think this was left over from previous iterations. It's now been removed for all the tests.